projects/eslint-plugin-experience-next/configs/recommended.ts

```diff
@@ -195,6 +248,10 @@ export default tseslint.config(
                     ],
                 },
             ],
+            '@taiga-ui/experience-next/no-implicit-public': 'error',
+            '@taiga-ui/experience-next/no-private-esnext-fields': 'error',
+            '@taiga-ui/experience-next/standalone-imports-sort': 'error',
+            '@taiga-ui/experience-next/strict-tui-doc-example': 'error',
```

Comment:
Is this rule specific only for Taiga Family projects ?

Reply:
fixed